lib/requirejs.js

```diff
@@ -409,8 +409,17 @@ function _initPackage(pkg, pkgs) {
         return false;
       }
     }
-    var configFile = path.resolve(pkg.path, pkg.bedrock.config);
-    require(configFile)(bedrock);
+    var configFiles = [];
+    if(pkg.bedrock.config instanceof Array) {
+      configFiles = pkg.bedrock.config.map(function(fileName) {
+        return path.resolve(pkg.path, fileName);
+      });
+    } else {
+      configFiles.push(path.resolve(pkg.path, pkg.bedrock.config));
+    }
+    configFiles.forEach(function(file) {
+      require(file)(bedrock);
+    });
```

Comment:
Could simplify this more:
```js
    var configFiles;
    if(pkg.bedrock.config instanceof Array) {
      configFiles = pkg.bedrock.config;
    } else {
      configFiles = [pkg.bedrock.config];
    }
    configFiles.forEach(function(file) {
      require(path.resolve(pkg.path, file))(bedrock);
    });
```
Sooo many lines, and `if`s are too scary, how about this:
```js
    var configFiles = (pkg.bedrock.config instanceof Array) ? pkg.bedrock.config : [pkg.bedrock.config];
    configFiles.forEach(function(file) {
      require(path.resolve(pkg.path, file))(bedrock);
    });
```
But wait, I just remembered `concat`:
```js
    var configFiles = [].concat(pkg.bedrock.config);
    configFiles.forEach(function(file) {
      require(path.resolve(pkg.path, file))(bedrock);
    });
```
And no reason for silly vars:
```js
    [].concat(pkg.bedrock.config).forEach(function(file) {
      require(path.resolve(pkg.path, file))(bedrock);
    });
```
If you love ES6, could one line it:
```js
    [].concat(pkg.bedrock.config).forEach(file => require(path.resolve(pkg.path, file))(bedrock));
```